user/docs/faq.md

```diff
@@ -0,0 +1,158 @@
+---
+sidebar_position: 4
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Troubleshooting & FAQ
+
+This page lists some common problems when installing or running NeoForge modpacks. If your problem isn't covered here, please see the [Getting Support][support] section.
+
+## Installing Java or NeoForge
+
+### I downloaded the installer file, but I have no way of running it!
+
+Please see the [Installing Java][installjava] section. Yes, even if you already have Minecraft: Java Edition installed.
+
+### I installed Java, but I'm still getting the wrong version!
+
+This likely means that your PATH variable is not set, or set incorrectly. The PATH variable is a system variable that tells your computer where to find Java. To fix it, do the following depending on your operating system:
+
+<Tabs defaultValue="windows">
+  <TabItem value="windows" label="Windows">
+Download and run the [Jarfix program][jarfix].
+  </TabItem>
+  <TabItem value="macos" label="MacOS">
+Open Finder. In Finder, open the Applications/Utilities folder and double-click Terminal.
+
+Run the following commands:
+
+- `echo export "JAVA_HOME=\$(/usr/libexec/java_home)" >> ~/.zprofile`
+- `echo export "PATH=$PATH:$JAVA_HOME/bin" >> ~/.zprofile`
+
+Then, close Terminal and try again.
+  </TabItem>
+  <TabItem value="linux" label="Linux">
+Open the terminal for your Linux distribution. Common names would be `GNOME Terminal` or `Konsole`, however it may vary depending on your exact setup.
```

Comment:
Maybe you should mention that some distros have specific ways to manage Java versions. (I'm thinking specifically Arch and Fedora with `archlinux-java` and `alternatives` respectively)

Reply:
I linked to the docs for that, I think mentioning this here would be scope-creep due to the tons of distros out there. (Linux ecosystem my belovehated)